reports/2025-07-13-Q2-report.md

```diff
@@ -0,0 +1,106 @@
+# Haskell Security Response Team - 2025 April–June report
+
+The Haskell Security Response Team (SRT) is a volunteer organization
+within the Haskell Foundation that is building tools and processes
+to aid the entire Haskell ecosystem in assessing and responding to
+security risks.  In particular, we maintain a [database][repo] of
+security advisories that can serve as a data source for security
+tooling.
+
+This report details the SRT activities for April–June 2025.
+
+[repo]: https://github.com/haskell/security-advisories
+
+The SRT is:
+
+- Fraser Tweedale
+- Gautier Di Folco
+- Lei Zhu
+- Mihai Maruseac
+- Montez Fitzpatrick
+- Tristan de Cacqueray
+
+
+## How to contact the SRT
+
+For assistance in coordinating a security response to newly
+discovered, high impact vulnerabilities, contact
+`security-advisories@haskell.org`.  Due to limited resources, we can
+only coordinate embargoed disclosures for high impact
+vulnerabilities affecting current versions of core Haskell tools and
+libraries, or in other exceptional cases.
+
+You can submit lower-impact or historical vulnerabilities to the
+advisory database via a pull request to our [GitHub
+repository][repo].
+
+You can also contact the SRT about non-advisory/security-response
+topics.  We prefer public communication where possible.  In most
+cases, [GitHub issues][gh-new-issue] are an appropriate forum.  But
+the mail address is there if no other appropriate channel exists.
+
+[gh-new-issue]: https://github.com/haskell/security-advisories/issues/new/choose
+
+
+## ZuriHac trip report
+
+Members of the team met in person at ZuriHac. We discussed handling
+package namespaces to support external registries as part of the
+advisories we manage (see [issue#240][issue-240]). With the help of
+other attendees, we re-discovered an issue with regard to cabal-install
+dependency confusion which lead to a new historical advisory:
+[HSEC-2025-0005][hsec-2025-005].
+
+The team also discussed long term project ideas to improve the ecosystem
+security. We have a few list scattered in our meeting notes and
+we'll collect the ideas in a top level file to be shared with the
+community.
+
+We would like to thank the ZuriHac organizers for the opportunity to
+meet with the other members of the ecosystem.
+
+[issue-240]: https://github.com/haskell/security-advisories/issues/240
+[hsec-2025-005]: https://github.com/haskell/security-advisories/pull/281
+
+
+## Advisory database
+
+1 contemporary advisory and no historical advisories were published
+during the reporting period.
+
+2 HSEC IDs (HSEC-2024-0004 and HSEC-2024-0005) **remain** reserved
```

Comment:
Does it make sense for vulnerabilities to be embargoed for a year? If a maintainer remains unresponsive, it's probably better to inform users about a vulnerability sooner than later.

Reply:
Well, the vulnerability is **already** impacting the whole community. It's just that the community does not know about it and cannot mitigate on its side. I mean, "do not use Haskell" **is** a plausible mitigation, if the vulnerability is severe enough.

(I have no inside knowledge of the topic FWIW)

Reply:
I would say if there are workarounds then we can publish, but if there are no workarounds then publishing is not responsible